### ISSUE TITLE
Controller manager prevents multiple robots from spawning

### ISSUE THREAD

**Issue by CSharpRon** [FlaSpaceInst/ezrassor_sim_description#4]
The controller_manager defined in ```config/controllers.yaml``` has the namespace locked to `ezrassor` at the top of the file. This prevents multiple robots from spawning since the controller_manager is always going to spawn the service at a namespace of /ezrassor. 

I tried to replace this value to `/**:` to see if it would take the namespace from the launch command, but that did not work. 

I would recommend posting this question on the ros forum and asking for help since there might be a config parameter that is missing.

Affected files:
- `launch/spawn_ezrassor.py`
- `config/controllers.yaml`

Replicating the issue:
```sh
# Bring up the sim
ros2 launch ezrassor_sim_gazebo gazebo_launch.py
ros2 launch ezrassor_sim_description spawn_ezrassor.py robot_name:=ezrassor
ros2 launch ezrassor_sim_description spawn_ezrassor.py robot_name:=ezrassor2 x:=10

# Now that both robots are spawned...
ros2 topic pub /ezrassor/wheel_instructions geometry_msgs/msg/Twist "{linear: {x: 200.0, y: 0.0, z: 0.0}, angular: {x: 200.0, y: 0.0, z: 0.0}}"
ros2 topic pub /ezrassor2/wheel_instructions geometry_msgs/msg/Twist "{linear: {x: 200.0, y: 0.0, z: 0.0}, angular: {x: 200.0, y: 0.0, z: 0.0}}"
```

**Comment by Nate8888** [comment]
This was a rather obscure issue. The current solution is a "hack" and it basically consists of duplicating the controllers with different node names. Ex: ```ezrassor```, ```ezrassor2```, ```ezrassor3```, ```ezrassor4```, and ```ezrassor5```.

**Comment by Nate8888** [comment]
Issue was fixed with #7
